src/test/java/org/opentripplanner/raptor/_data/RaptorTestConstants.java

```diff
@@ -27,6 +27,13 @@ public interface RaptorTestConstants {
   int D20m = durationInSeconds("20m");
   int D24h = durationInSeconds("24h");
 
+  /**
+   * There are 86400 seconds in a "normal" day(24 * 60 * 60). This is used for testing, logging
+   * and debugging, but does not base any important logic on this. A day with changes in
```

Comment:
```suggestion
   * and debugging, but don't base any important logic on this. A day with changes in
```

Reply:
It is not recommended to use apostrophized (or in any way) shortened words in professional or formal english. The doc was appropriate while it was part of the main code, but I moved it into the test code to make it impossible to use in the main code. So, I do not think the doc is needed any more. I will clean it up.